src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double baseSide;
+    private double equalSides;
+    private double topSide;
+
+    public IsoscelesTrapezoid(double baseSide, double equalSides, double topSide, String color) {
```

Comment:
It is better to use a constructor with no parameters. Instead, use setters to set the properties.